src/pages/Home.test.jsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen, waitFor, fireEvent } from '@testing-library/react'
+import {rest} from 'msw'
+import { server } from '../mocks/server.js'
+import { Home } from './Home'
+
+const setup = () => render(<Home />)
+
+describe('Home', () => {
+  it('should pick today\'s date as default and get and apod', async () => {
+    setup()
+
+    const today = Intl.DateTimeFormat("en-CA").format(Date.now())
+
+    const datePicker = screen.getByLabelText(/pick a date:/i)
+
+    expect(datePicker).toHaveValue(today)
+
+    await waitFor(() => screen.getByRole('heading'))
+
+    const apodTitle = screen.getByRole('heading')
```

Comment:
there is a selector called `findBy...` which can be used instead of the `waitFor + getBy`, something like this:
```js
await waitFor(() => screen.getByRole('heading'))
const apodTitle = screen.getByRole('heading')
```

can be changed to:

```js
const apodTitle = await screen.findByRole('heading')
```

Reply:
super cool! i did not know this